modules/custom/az_migration/migrations/az_exclude_node_title.yml

```diff
@@ -0,0 +1,52 @@
+id: az_exclude_node_title
+label: Exclude node title config
+migration_group: az_migration
```

Comment:
Doesn't the filename need to be `migrate_plus....yml` for the group functionality to work?

Reply:
no